source/org/zfin/marker/agr/GeneLinkMLInfo.java

```diff
@@ -69,6 +70,7 @@ public List<org.alliancegenome.curation_api.model.ingest.dto.GeneDTO> getAllGene
                 dto.setDataProviderDto(dataProvider);
                 dto.setTaxonCurie(ZfinDTO.taxonId);
                 dto.setModEntityId("ZFIN:" + marker.getZdbID());
+                dto.setGeneTypeCurie(marker.getSoTerm().getOboID());
                 GregorianCalendar date = ActiveData.getDateFromId(marker.getZdbID());
                 dto.setDateCreated(format(date));
```

Comment:
future ticket: add constraints to our db so we don't get aliases with empty values

Reply:
Sone